cylc/flow/workflow_files.py

```diff
@@ -315,6 +317,20 @@ class RemoteCleanQueueTuple(NamedTuple):
 * ssh -n "%(host)s" kill %(pid)s   # final brute force!
 """
 
+SUITERC_DEPR_MSG = (
+    f"The filename '{WorkflowFiles.SUITE_RC}' is deprecated "
+    f"in favour of '{WorkflowFiles.FLOW_FILE}'"
+)
+
+NO_FLOW_FILE_MSG = (
+    f"no {WorkflowFiles.FLOW_FILE} or {WorkflowFiles.SUITE_RC} "
+    "in {}"
+)
+
+REG_CLASH_MSG = (
+    "Workflow files found in both {0} and {1}. This command will use {0}."
+)
```

Comment:
This message could be a little clearer.

Reply:
What do you have in mind?

I suppose I can change `{1}` from being `/home/blah/blah/user/cylc-run/<run_dir>` to just `~/cylc-run/<run_dir>`, but I'm guessing that's not what you mean

Reply:
Not sure I had anything particular in mind.

I think "Workflow files" surprised me a little, might not be so obvious to a user that this implies a reference ambiguity between a src and run dir.

```
{input} could mean {src_dir} or ~/cylc-run/{run_dir.relative_to(cylc_run_dir)}, using {src_dir}
```

> I suppose I can change {1} from being /home/blah/blah/user/cylc-run/<run_dir> to just ~/cylc-run/<run_dir>

Would be good.